src/sentry/static/sentry/app/utils/projects.tsx

```diff
@@ -401,7 +401,7 @@ type FetchProjectsOptions = {
   prevSearch?: State['prevSearch'];
 } & Pick<Props, 'limit' | 'allProjects'>;
 
-async function fetchProjects(
+export async function fetchProjects(
```

Comment:
This is not intended to be exposed, can you explain a bit about what you're trying to do with projects?  Fetching all projects is not ideal as it will be painful for large orgs.

Reply:
Hrm I need to be able to know the counts of projects depending on the counts (but only so far as whether its >10 or not). so when a query is for all projects, whether that's 15 projects, or just 5 for an org.

Reply:
😬 would it be overkill to add an API that returns 2 values, # of projects in org and # of projects you're a member of? Otherwise we could be transferring/serializing thousands of projects for just the length.

Reply:
Thinking about it more, that's the better solution I think :+1: I'll flip the use of fetchProjects over to a different function/endpoint